timlubanga/ireport_web
# [Finishes #162109467]add login page

What does this PR do?
enables a user to  login the the application

Description of Task to be completed?

Enter username
Enter password


How should this be manually tested
Clone ireport_web repo into your computer, open the ui folder and run login.html file using any browser . The login page should open
**Any background context you want to provide?
This is a mockup. Full functionality is yet to be implmented
What are the relevant pivotal tracker stories?
162109467
![screenshot from 2018-11-24 05-32-45](https://user-images.githubusercontent.com/40349882/48963758-820a6c00-efaa-11e8-9d09-08f75b879ce4.png)
